test/unittest/tests/block/test_block_addr.cpp

```diff
@@ -0,0 +1,174 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+// This file unit tests block manager functions relating to packing and unpacking address cookies.
+
+#include "wt_internal.h"
+#include <catch2/catch.hpp>
+#include <iostream>
+#include <vector>
+
+void
+unpack_addr_cookie_and_check(const uint8_t *packed, uint32_t block_allocsize, wt_off_t pack_offset,
+  uint32_t pack_size, uint32_t pack_checksum)
+{
+    uint64_t o = 0, s = 0, c = 0;
```

Comment:
Can we call these offset, size, checksum? The addr code uses the shortenings but only to it's detriment.